playground/index.html

```diff
@@ -32,8 +32,8 @@
             <pre id="blockexplorer"></pre>
         </div>
         <div id="tab-importexport">
-            Import/Export
-            <input id="projectId" value="119615668" />
+            Import/Export<br />
+            Project ID: <input id="projectId" value="" />
```

Comment:
Could we leave in 119615668 as the default value, to make it easy to load an example project?